### ISSUE TITLE
Better colors for conflict solving

### ISSUE THREAD

**Issue by g7fernandes** [mrworkman/papercolor-vscode-redux#2]
When solving conflict, the background colors for current and incoming code are too dark. 

I recommend:

```
        "merge.currentHeaderBackground": "#398d3d",
        "merge.currentContentBackground": "#78c2a354",
        "merge.incomingHeaderBackground": "#2d8eb4",
        "merge.incomingContentBackground": "#63aad356",
```

**Comment by mrworkman** [comment]
You're right. I've always meant to go back to fix that. I will try to fix it soon.

**Comment by mrworkman** [comment]
Fixed. Released as v0.9.0. Sorry for the delay!
